qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -201,9 +201,7 @@ def run(self, dag):
                     f"specified) target basis: {list(expanded_target)}. This likely means the "
                     "target basis is not universal or there are additional equivalence rules "
                     "needed in the EquivalenceLibrary being used. For more details on this "
-                    "error see: "
-                    "https://qiskit.org/documentation/stubs/qiskit.transpiler.passes."
-                    "BasisTranslator.html#translation_errors"
+                    "error see: `translation_errors`_"
```

Comment:
```suggestion
```suggestion
                "error see: "
                "https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.transpiler.passes."
                "BasisTranslator#translation-errors"
```
```

Reply:
I checked the link and added a new commit!